### DISCUSSION TITLE
Scenario "Saint John": what is writing to this log file?

### DISCUSSION THREAD

**Discussion by fduran** [SadServers/sadservers#17] [Scenarios]
Discussions about the [Saint John scenario](https://github.com/fduran/sadservers/tree/main/scenarios/saintjohn)

**Comment by WuzenLiz** [comment]
ehm i've terminated the process and delete the bad.log file. Did i do it wrong?

**Reply by WuzenLiz** [reply]
Well, i've created bad logfile again and it correct.

**Reply by fduran** [reply]
Hello @WuzenLiz , thanks for your comment. You only need to terminate the process.

Some users were deleting the file  and passing the old check.sh validation script but the [new one](https://github.com/fduran/sadservers/tree/main/scenarios/saintjohn#test) will fail if you just delete the file.

**Comment by jonaswestphal** [comment]
what about the crontab entry 👁️

**Reply by fduran** [reply]
Good catch. That's a lazy way to get the program to start on reboot, otherwise I'd have to write a systemd file :-)
